perun-core/src/test/java/cz/metacentrum/perun/core/api/AuthzResolverIntegrationTest.java

```diff
@@ -26,6 +26,7 @@
 import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
```

Comment:
Not sure why added if there is no other change in the tests.

Reply:
Sorry I will remove it. I've just run some tests to be sure that everything works and I forgot to remove the imports after that. I did remove tests because they depended on the actual state of the config file which can change in the future.